tools/deb/istio-auth-node-agent.service

```diff
@@ -3,7 +3,7 @@ Description=istio-auth-node-agent: The Istio auth node agent
 Documentation=http://istio.io/
 
 [Service]
-ExecStart=/usr/local/bin/istio-auth-node-agent --logtostderr
+ExecStart=/usr/local/istio/bin/node-agent --logtostderr
```

Comment:
Okay, this change is new. @kyessenov do we even have istio directory for pilot agents ? i thought we always had them in /usr/local/bin..

Reply:
No, `/usr/local/blah` sounds odd to me. In terms of impact, these are just entry points in containers, so we can shuffle things if needed.

Reply:
the point is in a VM we shouldn't put stuff with generic names (istioctl is fine, it's ours) in /usr/local/bin - this addresses that (and is unrelated to pilot-agent which can also be in the same directory in another PR but also doesn't have to be)

Reply:
remember this is not containers folks, this is someone's actual pre-existing machine